tests/console/zypper_info.pm

```diff
@@ -30,11 +30,20 @@ sub run() {
     die "Missing package name. Expected: /$expected_package_name_vim/"
       unless $info_output_vim =~ /$expected_package_name_vim/;
 
-    if (check_var('DISTRI', 'sle')) {
+    my $is_opensuse = check_var('DISTRI', 'opensuse');
+    my $is_at_leat_sle_12_SP3 = check_var('DISTRI', 'sle') && (check_var('VERSION', '12-SP3') || int get_var('VERSION') >= 13);
+
+    if ($is_at_leat_sle_12_SP3) {
         # use dvd2 as the src-repository
         zypper_call('ar --disable --type plaindir cd:///?devices=/dev/sr1 repo-source');
     }
 
+    if ($is_opensuse || $is_at_leat_sle_12_SP3) {
```

Comment:
ISO_2 is set up in test module, which is executed by both. so the var ISO_2 would exist in SP2 making the test to fail.